### ISSUE TITLE
Default Sort on Related

### ISSUE THREAD

**Issue by johnpaulmedina** [Naxon/nova-field-sortable#7]
I was writing something to extend the default sort when I came across someone linking to your package. I noticed that you are only attaching the sort on the index and it is not able to sort on the related content.

Any plans on doing this or should I write my own?

**Comment by Naxon** [comment]
Added this to future plans. In the meantime you're more than welcome to make a PR.

**Comment by gauravmak** [comment]
@johnpaulmedina We can make this work with `HasMany` and `MorphMany` relationships already.

To make sure that sorting is done among _groups_ only, we can override [`buildSortQuery()` method](https://github.com/spatie/eloquent-sortable/blob/a30069224f08b125efe5b6962208fa8e75bd5f9e/src/SortableTrait.php#L234) on the model.

```php
/**
 * Build eloquent builder of sortable. (For spatie/eloquent-sortable package)
 *
 * @return \Illuminate\Database\Eloquent\Builder
 */
public function buildSortQuery()
{
    return static::query()->where('post_id', $this->post_id);
}
```

**Comment by timbroder** [comment]
@gauravmak do you mean you have the UI (the arrows) showing up in the `HasMany` grouping? 

If so, how? Thanks!

**Comment by gauravmak** [comment]
@timbroder Yes, that was the case as far as I can remember. I was displayed only the records of a group and then sorting had to work among the group records only.
